controllers/common/util.go

```diff
@@ -156,3 +156,12 @@ func GetResourcesDynamically(ctx context.Context, dynamic dynamic.Interface, gro
 
 	return list.Items, nil
 }
+
+func GetFirstNCharacter(str string, n int) string {
+	v := []byte(str)
```

Comment:
is there a reason to convert to []byte? Why not just compare with length of string, `len(str)`?